--- result/app.py	1969-12-31 19:00:01.000000000 -0500
+++ app.py	2022-11-29 08:58:46.043003572 -0500
@@ -474,7 +474,7 @@
             return redirect(url_for('index'))
 
         username = creds["username"]
-        account_id = creds["id"]
+        account_id = str(creds["id"])
 
         bridge = db.session.query(Bridge).filter_by(mastodon_account_id=account_id, mastodon_host_id=masto_host.id).first()
 
